client/my-sites/site-settings/feed-settings/index.jsx

```diff
@@ -0,0 +1,106 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+
+import Button from 'components/button';
+import CompactCard from 'components/card/compact';
+import CompactFormToggle from 'components/forms/form-toggle/compact';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormSettingExplanation from 'components/forms/form-setting-explanation';
+import FormTextInput from 'components/forms/form-text-input';
+import SectionHeader from 'components/section-header';
+import { getSelectedSiteId } from 'state/ui/selectors';
+
+class FeedSettings extends Component {
+	render() {
+		const {
+			fields,
+			handleSubmitForm,
+			handleToggle,
+			isRequestingSettings,
+			isSavingSettings,
+			onChangeField,
+			translate,
+		} = this.props;
+
+		const isDisabled = isRequestingSettings || isSavingSettings;
+
+		if ( 'undefined' === typeof fields.posts_per_rss ) {
+			// Do not allow these settings to be updated if they cannot be read from the API.
+			return null;
+		}
+
+		return (
+			<div className="feed-settings">
+				<SectionHeader label={ translate( 'Feed Settings' ) }>
+					<Button compact primary disabled={ isDisabled } onClick={ handleSubmitForm }>
+						{ isSavingSettings ? translate( 'Saving…' ) : translate( 'Save Settings' ) }
+					</Button>
+				</SectionHeader>
+				<CompactCard>
+					<FormFieldset>
+						{ translate( 'Display {{field /}} most recent blog posts', {
+							components: {
+								field: (
+									<FormTextInput
+										name="posts_per_rss"
+										type="number"
+										step="1"
+										min="0"
+										id="posts_per_rss"
+										value={
+											'undefined' === typeof fields.posts_per_rss ? 10 : fields.posts_per_rss
```

Comment:
Is this redundant given L38? I guess there's no harm leaving it, regardless.

Reply:
Good catch, I'll update it before I merge.